### ISSUE TITLE
Rename primitive in mesh/primitive to mode

### ISSUE THREAD

**Issue by glome83** [KhronosGroup/glTF#341]
There's a property named "primitive" in "mesh/primitive". This can lead to confusion. Rename it to "mode" as it is called in glDrawArrays etc.

**Comment by fabrobinet** [comment]
@glome83 Thanks for bringing this to our attention, so as here:
```json
      "primitives": [
                {
                    "attributes": {
                        "NORMAL": "accessor_25",
                        "POSITION": "accessor_23",
                        "TEXCOORD_0": "accessor_27"
                    },
                    "indices": "accessor_21",
                    "material": "blinn3-fx",
                    "primitive": 4
                }
            ]
```
you suggest instead:
```json
      "primitives": [
                {
                    "attributes": {
                        "NORMAL": "accessor_25",
                        "POSITION": "accessor_23",
                        "TEXCOORD_0": "accessor_27"
                    },
                    "indices": "accessor_21",
                    "material": "blinn3-fx",
                    "mode": 4
                }
            ]
```

Which I don't have a strong opinion about this but I undertand that it could be less confusing - only concern here is I don't find `mode` particularly explicit but it is true that is matches the GL naming of arguments and since the value is a glEnum it's fine... 

@pjcozzi what do you think ?

**Comment by pjcozzi** [comment]
This probably better since "primitives/primitive" isn't used in two different contexts.  Should I update the spec?

**Comment by fabrobinet** [comment]
@pjcozzi yeah, let's do it for 0.9

**Comment by pjcozzi** [comment]
Updated the spec in 5bcf79ee90b72b3a8a3fbc0ca2f43b20c82ab1fd in the `spec-9` branch.

**Comment by pjcozzi** [comment]
This can be closed when 1.0 is merged...or now.

**Comment by pjcozzi** [comment]
This is done in the `spec-1.0` branch.
